torchgeo/datasets/levircd.py

```diff
@@ -137,7 +137,7 @@ def _load_files(
         """
         files = []
         images = glob.glob(os.path.join(root, directory, split, "A", "*.png"))
-        images = sorted([os.path.basename(image) for image in images])
+        images = sorted(os.path.basename(image) for image in images)
```

Comment:
Also not really a fan of taking away the list comprehension brackets

Reply:
I agree with you on this one, I'm actually surprised this works.